src/Compatibility/Microsoft.DotNet.ApiCompatibility/Abstractions/Filtering/AccessibilityFilter.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.DotNet.ApiCompatibility.Abstractions
+{
+    internal class AccessibilityFilter : ISymbolFilter
```

Comment:
Looks to me like this is an externally visible filter?  Probably needs a better name.

Reply:
Sure I can rename. How about, `SymbolAccessibilityBasedFilter`?